Tests/Functional/RunnerTest.php

```diff
@@ -830,28 +855,245 @@ public function testExecutorNoStoreState()
+            ->getMock();
+        $loggersServiceStub->expects($this->any())
+            ->method('getLog')
+            ->will($this->returnValue($log));
+        $loggersServiceStub->expects($this->any())
+            ->method('getContainerLog')
+            ->will($this->returnValue($containerLogger));
+
+        /** @var JobMapper $jobMapperStub */
+        $jobMapperStub = $this->getMockBuilder(JobMapper::class)
+            ->disableOriginalConstructor()
+            ->getMock()
+        ;
+
+        $encryptorFactory = new ObjectEncryptorFactory(
+            'alias/dummy-key',
+            'us-east-1',
+            hash('sha256', uniqid()),
+            hash('sha256', uniqid())
+        );
+        $encryptorFactory->setComponentId('keboola.r-transformation');
+        $encryptorFactory->setProjectId($tokenInfo["owner"]["id"]);
+
+        /** @var StorageApiService $storageServiceStub */
+        /** @var LoggersService $loggersServiceStub */
+        $runner = new Runner(
+            $encryptorFactory,
+            $storageServiceStub,
+            $loggersServiceStub,
+            $jobMapperStub,
+            "dummy",
+            RUNNER_COMMAND_TO_GET_HOST_IP,
+            RUNNER_MIN_LOG_PORT,
+            RUNNER_MAX_LOG_PORT
+        );
+
+        $component = new Components($this->client);
+        try {
+            $component->deleteConfiguration('docker-demo', 'test-configuration');
+        } catch (ClientException $e) {
+            if ($e->getCode() != 404) {
+                throw $e;
+            }
+        }
+        $configuration = new Configuration();
+        $configuration->setComponentId('docker-demo');
+        $configuration->setName('Test configuration');
+        $configuration->setConfigurationId('test-configuration');
+        $configuration->setState(json_encode(['foo' => 'bar']));
+        $configData = [
+            'parameters' => [
+                'script' => [
+                    'import json',
+                    'with open("/data/out/state.json", "w") as state_file:',
+                    '   json.dump({"baz": "fooBar"}, state_file)',
+                ],
+            ],
+            'processors' => [
+                'after' => [
+                    [
+                        'definition' => [
+                            'component'=> 'keboola.processor-dumpy',
+                        ],
+                        'parameters' => [
+                            'script' => [
+                                'import json',
+                                'from shutil import copyfile',
+                                'try:',
+                                '   copyfile("/data/in/state.json", "/data/out/files/state.json")',
+                                '   with open("/data/out/files/state.json.manifest", "w") as state_file_manifest:',
+                                '       json.dump({"tags": ["test-state", "docker-bundle-test"]}, state_file_manifest)',
```

Comment:
nemohl by to ten procesor vyblejt na stdout a pak si zkontrolovat message v loghandleru?

Reply:
nemohl, protoze stdout procesoru se zahazuje

Reply:
má to nějakej důvod? občas mě napadlo, že by mohl vypisovat, co dělá...

Reply:
hm, to je dobra otazka!

Reply:
ha, tak output z procesoru normalne funguje, ja to predtim zkousel, ale mel jsem tam blbe testhandler a pak jsem to nejak prestal zkoumat, takze takhle je to snadnejsi

Reply:
super!